.github/workflows/docs.yml

```diff
@@ -0,0 +1,55 @@
+name: docs
+
+on:
+  push:
+    branches:
+     - 'main'
+
+jobs:
+  update-docs:
+    if: github.repository == 'symforce-org/symforce'
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install build dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y \
+            build-essential \
+            doxygen \
+            libgmp-dev \
+            pandoc
+
+      - name: Install python dependencies
+        run: pip install -r dev_requirements.txt
+
+      - name: Run cmake build
+        run: |
+          cmake -B build \
+            -DSYMFORCE_BUILD_OPT=ON \
+            -DSYMFORCE_BUILD_CC_SYM=ON \
+            -DSYMFORCE_BUILD_EXAMPLES=OFF \
+            -DSYMFORCE_BUILD_TESTS=OFF \
+            -DSYMFORCE_BUILD_SYMENGINE=ON \
+            -DSYMFORCE_GENERATE_MANIFEST=ON \
+            -DSYMFORCE_BUILD_BENCHMARKS=OFF
+          cmake --build build -j $(nproc)
+
+      - name: Build docs
+        run: |
+          pip install build/lcmtypes/python2.7
+          make docs
+
+      - name: Deploy docs
+        uses: cpina/github-action-push-to-another-repository@v1.5
+        env:
+          SSH_DEPLOY_KEY: ${{ secrets.SSH_DOCUMENTATION_DEPLOY_KEY }}
+        with:
+          source-directory: build/docs
+          destination-github-username: 'symforce-org'
+          destination-repository-name: 'symforce-org.github.io'
+          user-email: bradley.solliday@skydio.com
+          commit-message: Regenerate docs from $GITHUB_REF
```

Comment:
Not sure what we want the commit message to look like. Currently, this will look like `Regenerate docs from refs/heads/bradley/auto_deploy_docs_2` (for example).

Reply:
Would be nice to have the symforce commit sha maybe?  But this is also fine

Reply:
Okay. I changed it to be the sha instead (not the sha of the branch you're merging, but the one that appears in main)